Client/Frontend/UserContent/UserScripts/FingerprintingProtection.js

```diff
@@ -13,15 +13,7 @@ if (webkit.messageHandlers.$<handler>) {
         function installIFrameMethodTrap(obj, method) {
             let orig = obj[method];
             obj[method] = function () {
-                var last = arguments[arguments.length - 1]
-                if (last && last.toLowerCase() === 'canvas') {
-                    // Prevent fingerprinting using contentDocument.createElement('canvas'),
-                    // which evades trapInstanceMethod when the iframe is sandboxed
-                    sendMessage({ obj: `${obj}`, method: method })
-                } else {
-                    // Otherwise apply the original method
-                    return orig.apply(this, arguments)
-                }
+                return orig.apply(this, arguments)
```

Comment:
You can remove this entire hook. All it is doing is calling the original method.

Reply:
@Brandon-T thanks for the heads up -- I removed the `installIFrameMethodTrap` altogether.